### PR TITLE
remove remaining text when the path include the occurences

### PR DESCRIPTION
**Pull request by robUx4**
Adressing a comment from Benjamin Kaduk https://mailarchive.ietf.org/arch/msg/cellar/epuOIwz8VXiu3V9-AbHNa7BRZjY

### CODE REVIEW

File: specification.markdown [hunk 1]
```diff
@@ -737,7 +735,7 @@ This document contains definitions of all EBML Elements of the EBML Header.
 
 name: EBML
 
-path: `1*1(\EBML)`
+path: `\EBML`
```
Comment (dericed) [inline specification.markdown]: I don't understand this change and the subsequent ones. If the PathMinOccurrence is not present then it is understood to be 0, so this edit seems to mean that the EBML Header is no longer required.

**Review by dericed — COMMENTED** [review]
This commit seems to remove most uses of PathMinOccurrence and PathMaxOccurrence but I don't understand what replaces that. Also if such edits to the `Items` xml is occurring, then the same edits should effect the ebml_schema_example.xml file.

File: specification.markdown (hunk 1 above)
Comment (robUx4) [reply specification.markdown]: `PathMinOccurrence` and `PathMaxOccurrence` are only use for global elements which `\EBML` and the following ones are not:
> PathMinOccurrence represents the minimum number of element path required between the EBMLFixedParent and the Global Element EBMLElementPath.
> PathMaxOccurrence represents the maximum number of element path possible between the EBMLFixedParent and the Global Element EBMLElementPath

I have to rework the global elements path to make this clearer (#326)

Basically, most pathes don't have occurences in them anymore. Only the Global Elements do (for now).

**Comment by dericed** [comment]
I suggest closing in favor of https://github.com/cellar-wg/ebml-specification/pull/331

**Comment by robUx4** [comment]
No objection although one can be done after the other (there won't be any conflict)

**Comment by dericed** [comment]
closing in favor of the work completed in https://github.com/cellar-wg/ebml-specification/pull/331

**Pull request closed without merging**